git-dependencies

```diff
@@ -62,10 +69,13 @@ def create_parser():
 									 description='Manage dependencies in git repositories',
 									 formatter_class=argparse.RawTextHelpFormatter)
 	parser.add_argument('-r', '--recursive', action='store_true', help='recurse into dependencies')
+	parser.add_argument('-o', '--overrides', help='override dependencies from file')
+	parser.add_argument('-oa', '--override-all', action='store_true', help='enforce complete override (fail if a dependency is not overridden)')
```

Comment:
It seems that this flag is hard to understand, maybe it should be negated (--allow-partial-override).
Or we drop the flag and have two separate override "modes", like --override-full=<file> and --override-partial=<file>